hm_1/tasks/task_08_01.py

```diff
@@ -0,0 +1,28 @@
+"""
+We have a file that works as key-value storage, each line is represented as key and value separated by = symbol, example:
+
+name=kek last_name=top song_name=shadilay power=9001
+
+Values can be strings or integer numbers. If a value can be treated both as a number and a string, it is treated as number.
+
+Write a wrapper class for this key value storage that works like this:
+
+storage = KeyValueStorage('path_to_file.txt') that has its keys and values accessible as collection items and as attributes. Example: storage['name'] # will be string 'kek' storage.song_name # will be 'shadilay' storage.power # will be integer 9001
```

Comment:
I'd recommend formatting this description in more readable by splitting it into short lines.

Some important parts of this task do not fit the screen.